SPECS/filesystem/filesystem.spec

```diff
@@ -1,817 +1,969 @@
-#
-#	7.10. Configuring the Linux Console"
-#
-cat > %{buildroot}/etc/sysconfig/console <<- "EOF"
-# Begin /etc/sysconfig/console
-#       Begin /etc/sysconfig/console
-#       KEYMAP="us"
-#       FONT="lat1-16 -m utf8"
-#       FONT="lat1-16 -m 8859-1"
-#       KEYMAP_CORRECTIONS="euro2"
-#       UNICODE="1"
-#       LEGACY_CHARSET="iso-8859-1"
-# End /etc/sysconfig/console
-EOF
-#
-#	7.13. The Bash Shell Startup Files
-#
-cat > %{buildroot}/etc/profile <<- "EOF"
-# Begin /etc/profile
-# Written for Beyond Linux From Scratch
-# by James Robertson <jameswrobertson@earthlink.net>
-# modifications by Dagmar d'Surreal <rivyqntzne@pbzpnfg.arg>
-
-# System wide environment variables and startup programs.
-
-# System wide aliases and functions should go in /etc/bashrc.  Personal
-# environment variables and startup programs should go into
-# ~/.bash_profile.  Personal aliases and functions should go into
-# ~/.bashrc.
-
-# Functions to help us manage paths.  Second argument is the name of the
-# path variable to be modified (default: PATH)
-pathremove () {
-        local IFS=':'
-        local NEWPATH
-        local DIR
-        local PATHVARIABLE=${2:-PATH}
-        for DIR in ${!PATHVARIABLE} ; do
-                if [ "$DIR" != "$1" ] ; then
-                  NEWPATH=${NEWPATH:+$NEWPATH:}$DIR
-                fi
-        done
-        export $PATHVARIABLE="$NEWPATH"
-}
-
-pathprepend () {
-        pathremove $1 $2
-        local PATHVARIABLE=${2:-PATH}
-        export $PATHVARIABLE="$1${!PATHVARIABLE:+:${!PATHVARIABLE}}"
-}
-
-pathappend () {
-        pathremove $1 $2
-        local PATHVARIABLE=${2:-PATH}
-        export $PATHVARIABLE="${!PATHVARIABLE:+${!PATHVARIABLE}:}$1"
-}
-
-export -f pathremove pathprepend pathappend
-
-# Set the initial path
-# Block unnessary as this is set elsewhere.
-# export PATH=$PATH:/bin:/usr/bin
-
-# if [ $EUID -eq 0 ] ; then
-#         pathappend /sbin:/usr/sbin
-#         unset HISTFILE
-# fi
-
-# Setup some environment variables.
-export HISTSIZE=1000
-export HISTIGNORE="&:[bf]g:exit"
-
-# Set some defaults for graphical systems
-export XDG_DATA_DIRS=/usr/share/
-export XDG_CONFIG_DIRS=/etc/xdg/
-
-# Setup a red prompt for root and a green one for users.
-NORMAL="\[\e[0m\]"
-RED="\[\e[1;31m\]"
-GREEN="\[\e[1;32m\]"
-if [[ $EUID == 0 ]] ; then
-  PS1="$RED\u@\h [ $NORMAL\w$RED ]# $NORMAL"
-else
-  PS1="$GREEN\u@\h [ $NORMAL\w$GREEN ]\$ $NORMAL"
-fi
-
-for script in /etc/profile.d/*.sh ; do
-        if [ -r $script ] ; then
-                . $script
-        fi
+rm -rf %{buildroot}
+mkdir %{buildroot}
+%if %{with locales}
+install -p -c -m755 %SOURCE2 %{buildroot}/iso_639.sed
+install -p -c -m755 %SOURCE3 %{buildroot}/iso_3166.sed
+%endif
+
+cd %{buildroot}
+
+Paths=(
+        afs boot dev \
+        etc/{X11/{applnk,fontpath.d,xinit/{xinitrc,xinput}.d},xdg/autostart,opt,pm/{config.d,power.d,sleep.d},skel,sysconfig,keys/ima,pki,bash_completion.d,rwtab.d,statetab.d} \
+        home media mnt opt root run srv tmp \
```

Comment:
@romoh just confirming that this change to delete the /srv symlink and have /srv be a real directory matches the expectation from this recent 2.0 change https://github.com/microsoft/azurelinux/pull/7687

Reply:
yes it does address that